includes/frontend.php

```diff
@@ -581,6 +590,11 @@ public function enqueue_styles() {
 		wp_enqueue_style( 'elementor-animations' );
 		wp_enqueue_style( 'elementor-frontend' );
 
+		if ( Plugin::instance()->is_legacy_mode_active ) {
```

Comment:
Couldn't you do the registration here as well?

Reply:
Needs to be available in the editor too no?